### DISCUSSION TITLE
Can I use colour on a string column?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#140] [Q&A] [answered]
Hello,

Firstly what a great plugin, thanks, we use it a lot!

I've imported a CSV that lists network devices and if they are On of Off.  I've set the column to 'String', but is there any way I can have the text show as green if On and red of Off?

Thanks

**Comment by marcusolsson** [comment, marked as answer]
You can, but data sources (like this one) typically don't control how data is visualized, so it's more of a Grafana question. You can use Thresholds and Color schemes in the panel editor, but how the color is used depends on the visualization you're using.

**Reply by G0nz0uk** [reply]
Got it all working thanks!
